dydx_withdrawal/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from dydx_withdrawal.views import Withdrawal
+
+urlpatterns = [
+    path("slow", Withdrawal.as_view({"post": "slow_withdrawal"})),
+    path("fast", Withdrawal.as_view({"post": "fast_withdrawal"})),
+    path("transfer", Withdrawal.as_view({"get": "transfer_info"})),
```

Comment:
"transfer" seems confusing cause we're actually not transferring anything, instead we're get transfer/withdrawal details..

so path should be transfer_info or transfer_details